js/src/ui/TxList/txList.css

```diff
@@ -42,10 +42,11 @@
     }
 
     &.timestamp {
-      padding-top: 1.5em;
-      text-align: right;
+      max-width: 5em;
+      padding-top: 0.75em;
+      text-align: center;
       line-height: 1.5em;
-      opacity: 0.5;
+      color: grey;
```

Comment:
Ahh, ok, the links as well, makes sense. Maybe be more specific then with `>span` (FormattedMessage)